DNN Platform/Library/Services/FileSystem/Providers/StandardFolderProvider.cs

```diff
@@ -193,13 +192,16 @@ public override string GetFileUrl(IFileInfo file)
         {
             Requires.NotNull("file", file);
 
-            var portalSettings = GetPortalSettings(file.PortalId);
+            var portalSettings = file.PortalId == PortalSettings.Current.PortalId ?
```

Comment:
```suggestion
            var portalSettings = file.PortalId == PortalSettings.Current?.PortalId ?
```

Reply:
It could be possible that this API is used from outside of an HTTP request, so that `PortalSettings.Current` might be `null`

Reply:
You are right @bdukes 
Fixed!